static/src/javascripts/projects/common/modules/spacefinder.ts

```diff
@@ -429,8 +429,9 @@ const getMeasurements = (
 			result[selector] = selectedElements.map(getDimensions);
 			return result;
 		}, {});
+
 		return {
-			bodyTop: bodyDims?.top ?? 0,
+			bodyTop: bodyDims?.top ? bodyDims.top + window.scrollY : 0,
```

Comment:
Would this mean that we only insert ads from the scroll position onwards i.e. not from the top of the page?

Reply:
Confusingly (well, I thought it was confusing), bodyDims.top is relative to the scroll position. The bottom example with the purple box displays it quite well: https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect

That's why we need to add the scroll position on to the bodyDims.top, to get the number of pixels distance between the top of the page and top of the body element.

I will refactor this a bit so hopefully it will make a bit more sense, or at the least, add a comment.

Reply:
Thats much clearer now thanks Dom